src/babel/transformation/transformers/es6/arrow-functions.js

```diff
@@ -1,4 +1,28 @@
+/**
```

Comment:
Hmm, I was kinda hoping that some day this might be used for generating the transformer documentation for the site.

I can clarify about shadow-functions below